python/samples/kernel-syntax-examples/chat_gpt_api.py

```diff
@@ -25,10 +29,12 @@
 
 api_key, org_id = sk.openai_settings_from_dot_env()
 kernel.add_service(
-    sk_oai.OpenAIChatCompletion(service_id="chat-gpt", ai_model_id="gpt-3.5-turbo", api_key=api_key, org_id=org_id)
+    sk_oai.OpenAIChatCompletion(
+        service_id="chat-gpt", ai_model_id="gpt-3.5-turbo", api_key=api_key)
 )
-
-settings = kernel.get_prompt_execution_settings_from_service(ChatCompletionClientBase, "chat-gpt")
```

Comment:
There is something for this, just needs the arguments the other way around and add _id: `settings = kernel.get_prompt_execution_settings_from_service_id("chat-gpt", ChatCompletionClientBase)`

Reply:
yes you are right, we can do this as well.

Reply:
could you please revert this change, I want the samples to be as simple and clear as possible!

Reply:
sure, am gonna revert it as earlier i did